### PR TITLE
[15.0][IMP] sale_order_line_menu: Added invoice lines in tree view

### PR DESCRIPTION
**Pull request by EmilioPascual**
Added invoice lines in tree view

@moduon MT-2028

image

image

### CODE REVIEW

**Comment by EmilioPascual** [comment]
@Shide can you review, please?

**Comment by Shide** [comment]
Another module has been added to the PR `sale_order_product_recommendation_secondary_unit`. I think it's not intended

File: sale_order_product_recommendation_secondary_unit/README.rst
Comment (Shide) [inline sale_order_product_recommendation_secondary_unit/README.rst:1]: Another module has been added to the PR `sale_order_product_recommendation_secondary_unit`. I think it's not intended

**Review by Shide — CHANGES_REQUESTED** [review]

File: sale_order_product_recommendation_secondary_unit/README.rst
Comment (EmilioPascual) [reply sale_order_product_recommendation_secondary_unit/README.rst:1]: You're right. Fixed

**Review by fcvalgar — APPROVED** [review]
Functional review,
LGTM, Thank you @EmilioPascual.

image

**Review by rafaelbn — APPROVED** [review]
/ocabot merge minor

**Pull request merged**